tests/ui/diagnostic_namespace/requires_path.stderr

```diff
@@ -0,0 +1,8 @@
+error: cannot find attribute `diagnostic` in this scope
+  --> $DIR/requires_path.rs:3:3
+   |
+LL | #[diagnostic]
+   |   ^^^^^^^^^^
```

Comment:
Before stabilization of the namespace, we'll want to customize this error to be something along the lines of "`#[diagnostic]` needs to be followed by one of `*`".